### PR TITLE
Calculating available memory from MemAvailable

### PR DESCRIPTION
**Pull request by rdkdd**
This is a part of avocado-framework/avocado-vt#948 PR.

Signed-off-by: Radek Duda <rduda@redhat.com>

### CODE REVIEW

File: avocado/utils/memory.py [hunk 1]
```diff
@@ -48,6 +48,13 @@ def memtotal():
     return read_from_meminfo('MemTotal')
 
 
+def availablememtotal():
+    """
+    Read ``MemAvailable`` from meminfo.
+    """
+    return read_from_meminfo('MemAvailable')
```
Comment (ldoktor) [inline avocado/utils/memory.py]: TBH I didn't noticed we have the `freememtotal` and `memtotal` but I don't think such methods should be present. The difference in test code is:

```python
memory.read_from_meminfo("MemAvailable")
```
vs.
```python
memory.availablememtotal()
```

Which is almost the same size but requires 7 lines in library. I'd suggest improving the docstring in `read_from_meminfo` by listing:

```python
    """
    ...
    Most important keys:
    MemTotal - Overall memory size
    MemAvailable - Currently available memory
    MemFree - Free memory
    ...
    """
```

and skip those rather simplistic methods.

Comment (ldoktor) [inline avocado/utils/memory.py]: Actually I'd even skip the docstring improvement. People can google-search for it...

Comment (rdkdd) [inline avocado/utils/memory.py]: Ok @ldoktor. I proposed this only for the sake of consistency with `MemTotal` and `freeMemTotal` methods. But generally agree with you - those methods are not necessary. I can close this PR and accordingly modify https://github.com/avocado-framework/avocado-vt/pull/948.

Comment (ldoktor) [inline avocado/utils/memory.py]: OK, thank you.

**Pull request closed without merging**